src/X.Extensions.Logging.Telegram/X.Extensions.Logging.Telegram.csproj

```diff
@@ -1,28 +1,31 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <LangVersion>default</LangVersion>
-        <Authors>Andrew Gubskiy</Authors>
-        <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
         <Title>X.Extensions.Logging.Telegram</Title>
-        <PackageLicenseUrl>https://github.com/ernado-x/X.Extensions.Logging.Telegram/blob/main/LICENSE</PackageLicenseUrl>
-        <PackageIconUrl>https://ru.gravatar.com/userimage/8071071/f2dc08ee7e4016451f64a7ae9cffd110.png?size=200</PackageIconUrl>
-        <PackageProjectUrl>https://github.com/ernado-x/X.Extensions.Logging.Telegram</PackageProjectUrl>
-        <RepositoryUrl>https://github.com/ernado-x/X.Extensions.Logging.Telegram</RepositoryUrl>
-        <PackageTags>telegram, logging, log, logs, tg</PackageTags>
         <Description>Telegram logging provider</Description>
-        <RepositoryType>git</RepositoryType>
-        <Version>1.0.6.1</Version>
-        <PackageVersion>1.0.6.1</PackageVersion>
-        <TargetFrameworks>net6.0;net7.0;netstandard2.0;netstandard2.1</TargetFrameworks>
+
+        <LangVersion>default</LangVersion>
+        <TargetFrameworks>net6.0;net8.0;netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Why so many targets? Actually, `netstandard2.0` core all this. Also, NuGet can substitute other supported versions

Reply:
Also, `X.Serilog...` doesn't support `netstandard2.0` and it's impossible to meet the requirements of the `netstandard`

Reply:
In general multi targeting library provide more streamline experience  for consumers on different .NET versions.

Regarding X.Extensions.Logging.Telegram – it can have more target frameworks than X.Extensions.Serilog.Sinks.Telegram The dependency tree for X.Extensions.Serilog.Sinks.Telegram will be built by utilizing only required platforms.

Reply:
My concern here is if we'd like to unify the codebase for some everyday tasks, it makes sense to reuse some pieces of code, and now it looks like it won't be so straightforward because of a wide range of targets of `X.Extensions.Logging.Telegram`, but I can make a mistake. Anyway, we can always cover them with a .NET version checks